packages/flutter_tools/lib/src/xcode_project.dart

```diff
@@ -146,6 +146,25 @@ class IosProject extends FlutterProjectPlatform implements XcodeBasedProject {
   /// Xcode workspace shared workspace settings file for the host app.
   File get xcodeWorkspaceSharedSettings => xcodeWorkspaceSharedData.childFile('WorkspaceSettings.xcsettings');
 
+  /// Do all plugins support arm64 simulators to run natively on an ARM Mac?
+  bool get pluginsSupportArmSimulator {
```

Comment:
Optional nit: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#getters-feel-faster-than-methods
Parsing a file from disk is probably on the heavy side for a getter